### ISSUE TITLE
Add README to demos folder

### ISSUE THREAD

**Issue by mmkal** [material-components/material-components-web#131]
There's a demos folder, with lots of useful-looking html, but I'm unsure how we're supposed to view it? I've tried cloning and simply opening the html files, but am getting 

`assets/material-components-web.css.js Failed to load resource: net::ERR_FILE_NOT_FOUND`

There's no .md file in the demos folder, no `assets` folder anywhere I could find, and no package.json so running npm install doesn't do much. Is it explained anywhere how to actually view these demos?

**Comment by amsheehan** [comment]
This is something we should address. In the mean time you can read the instructions on running the demo [here](https://github.com/material-components/material-components-web#running-the-demos)

TL;DR: After installing everything, run: `npm run dev`

**Comment by mmkal** [comment]
Got it, thanks - that works for me for now. I won't close this since it looks like you may plan to add some docs to the demos folder, but if that's tracked somewhere else feel free to close for me.

**Comment by traviskaufman** [comment]
Agreed that we should put a basic README in the demos folder. Changing the issue title and adding to tracker.
